cmd/csi-node-driver-registrar/main.go

```diff
@@ -69,7 +60,7 @@ var (
 	healthzPort             = flag.Int("health-port", 0, "(deprecated) TCP port for healthz requests. Set to 0 to disable the healthz server. Only one of `--health-port` and `--http-endpoint` can be set.")
 	httpEndpoint            = flag.String("http-endpoint", "", "The TCP network address where the HTTP server for diagnostics, including pprof and the health check indicating whether the registration socket exists, will listen (example: `:8080`). The default is empty string, which means the server is disabled. Only one of `--health-port` and `--http-endpoint` can be set.")
 	showVersion             = flag.Bool("version", false, "Show version.")
-	mode                    = flag.String("mode", ModeRegistration, `The running mode of node-driver-registrar. "registration" runs node-driver-registrar as a long running process. "kubelet-registration-probe" runs as a health check and returns a status code of 0 if the driver was registered successfully, in the probe definition make sure that the value of --kubelet-registration-path is the same as in the container.`)
+	mode                    = flag.String("mode", ModeRegistration, "DEPRECATED. This flag is no longer used. If this is set to kubelet-registration-probe, the driver will immediately exist successfully without registering with CSI.")
```

Comment:
```suggestion
	mode                    = flag.String("mode", ModeRegistration, "DEPRECATED. This flag is no longer used. If this is set to kubelet-registration-probe, the driver will immediately exit successfully without registering with CSI.")
```

Reply:
Well, it will immediately exist too... :-P